spring-rest-http/src/test/java/com/baeldung/requestmapping/FooMappingExamplesControllerTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.requestmapping;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(FooMappingExamplesController.class)
+public class FooMappingExamplesControllerTest {
```

Comment:
Rename as *UnitTest or *IntegrationTest (whichever is appropriate)

This should fix the Jenkins build

Reply:
I have added the "Unit" suffix in the following commit: `7458950e5697183446f28409260ae417f85fd859`